### PR TITLE
Removing deprecated opt in/out routes

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

While making some changes I noticed that there were some opt in/out routes that I don't think are needed anymore. This just cleans them up.
## What is the value of this and can you measure success?

Cleaning is always good ✨ 
## Does this affect other platforms - Amp, Apps, etc?

Nope!
## Request for comment

@desbo @guardian/dotcom-platform

### CODE REVIEW

File: applications/app/controllers/OptInController.scala [hunk 1]
```diff
@@ -29,16 +29,12 @@ class OptInController extends Controller {
 
   def handle(feature: String, choice: String) = Action { implicit request =>
     Cached(60)(WithoutRevalidationResult(feature match {
-      case "https" => https.opt(choice)
       case "hsts" => hsts.opt(choice)
```
Comment (NataliaLKB) [inline applications/app/controllers/OptInController.scala]: @desbo Is this opt in/out still used?

Comment (desbo) [inline applications/app/controllers/OptInController.scala]: I don't think so! @JustinPinner, do you know if anyone still uses the HTTPS opt-in/out routes?

Comment (philwills) [inline applications/app/controllers/OptInController.scala]: I don't believe that we need any of this any longer as everyone should always be on HTTPS.

HSTS is now also on generally, so I believe can also go, though @mchv might have an opinion.

**Comment by mchv** [comment]
👍 thanks @NataliaLKB!

**Pull request merged**